src/main/java/com/example/TicketBookingApp/domain/Reservation.java

```diff
@@ -0,0 +1,48 @@
+package com.example.TicketBookingApp.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder(toBuilder = true)
+public class Reservation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private LocalDate date;
+
+    @NotNull
+    private LocalTime time;
```

Comment:
1. Czemu nie zwinąć tego do wspólnego typu "datoczasowego"?
2. Jak rozumieć ten czas? To jest czas rezerwacji, czas filmu, czas wstawiania rekordu do bazy? Jak można by moją wątpliwość rozwiać?
3. Jakie jest ryzyko używania lokalnych datoczasów?

Reply:
Tutaj faktycznie jest mój ewidentny błąd. Powinno być jedno pole o nazwie _createReservationTime_ reprezentujące czas utworzenia rezerwacji. Używanie lokalnych datoczasów jest ryzykowne z racji występowania różnych stref czasowych.

Reply:
A pytanie nr 2?

Reply:
Na samym początku założyłem, że rezerwacja będzie na jeden seans, dlatego potraktowałem czas seansu i rezerwacji jako ten sam czas. Nie jest to dobre rozwiązanie i zmienna w klasie Reservation powinna być niezależna i reprezentować czas utworzenia rezerwacji. Dlatego powinno być to jedno pole o nazwie np createReservationTime.